includes/wikia/tests/UserPasswordTest.php

```diff
@@ -0,0 +1,207 @@
+<?php
+
+use Wikia\Service\Helios\ClientException;
+use Wikia\Service\Helios\HeliosClient;
+use Wikia\DependencyInjection\Injector;
+use Wikia\DependencyInjection\InjectorBuilder;
+
+class UserPasswordTest extends WikiaBaseTest {
+
+	const TEST_USER_ID = 4;
+
+	/** @var User */
+	private $testUser;
+
+	private $heliosClientMock;
+
+	protected static $currentInjector;
+
+	public static function setUpBeforeClass() {
+		self::$currentInjector = Injector::getInjector();
+	}
+
+	public function setUp() {
+		parent::setUp();
+		$this->setupAndInjectServiceMocks();
+
+		$this->testUser = User::newFromId( self::TEST_USER_ID );
+	}
+
+	public static function tearDownAfterClass() {
+		Injector::setInjector( self::$currentInjector );
+	}
+
+	private function setupAndInjectServiceMocks() {
+		$this->heliosClientMock = $this->getMock( HeliosClient::class,
+			[ 'login', 'forceLogout', 'invalidateToken', 'register', 'info', 'generateToken',
+				'setPassword', 'validatePassword', 'deletePassword' ] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "setPassword" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "validatePassword" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "deletePassword" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)